fracture_sets.py

```diff
@@ -445,15 +445,15 @@ def _define_centers_by_boxes(self, domain, distribution="poisson", intensity=Non
 
         # It is assumed that the intensities are computed relative to boxes of the
         # same size that are assigned in here
-        area_of_box = 1
+        area_of_box = self.domain_measure(domain) / (nx * ny)
 
         pts = np.empty(num_boxes, dtype=np.object)
 
         # First generate the full set of points with maximum intensity
         counter = 0
         for i in range(nx):
             for j in range(ny):
-                num_p_loc = stats.poisson(max_intensity * area_of_box).rvs(1)[0]
+                num_p_loc = stats.poisson(2 * max_intensity * area_of_box).rvs(1)[0]
```

Comment:
The intensity is set for the number of fractures, and each fracture has __two__ points

Reply:
ups, I realized this was a misktake. The line below adds both points